src/js/controller/captions.js

```diff
@@ -131,6 +131,10 @@ const Captions = function(_model) {
             }
         }
         // set the index without the side effect of storing the Off label in _selectCaptions
+        if (_model.get('enableDefaultCaptions') && _model.get('autostart')) {
```

Comment:
I think this is fine as long as we're fine with override a viewer's language choice when the current video does not have captions of that choice available